### ISSUE TITLE
Imbalanced dataset problems

### ISSUE THREAD

**Issue by ArlindKadra** [automl/Auto-PyTorch#195]
Currently, there are a few problems that crash the library when working with imbalanced datasets.

- There are Null values and one unique value in the train split and Null values in the test set. The library will try to convert the feature column to float on the test set and it will throw an error. This condition should be removed.
- There are Null values in the train split and Null and a few values at the test split, the other non-null values in the test set should be ignored so that the run does not crash.

**Comment by ravinkohli** [comment]
I'll leave this issue open for reference as it is still not fixed in the development branch
